assets/js/modules/analytics/datastore/settings.js

```diff
@@ -198,9 +219,10 @@ export function validateCanSubmitChanges( select ) {
 		INVARIANT_INVALID_ACCOUNT_ID
 	);
 
+	const isUAEnabled = select( CORE_FORMS ).getValue( FORM_SETUP, 'enableUA' );
```

Comment:
```suggestion
```

Looks like this is already defined on line 96: https://github.com/google/site-kit-wp/pull/6808/files#diff-a604176804435fc38d183c33dbd13d5cbd0f2b7cb4e892da7c38a0077dbd01a8R96

I think this can be removed, right?

Reply:
No, both are two different functions. Line 96 is part of `submitChanges` and line 222 is part of `validateCanSubmitChanges`.

Reply:
Ohhhh right on. 😅  The GitHub diff just made that really tough to see. Thanks! 🙏🏻